lib/scenic/adapters/postgres.rb

```diff
@@ -17,6 +17,10 @@ def self.drop_view(name)
         execute "DROP VIEW #{name};"
       end
 
+      def self.extensions(base = ActiveRecord::Base)
```

Comment:
In the interest of providing a consistent API for future expansion to other databases, do you think something like `additional_ignore_tables` would be better? Then the railtie doesn't have to change for databases that don't have a concept of `extensions`?